src/contribute/contribution-guide.md

```diff
@@ -264,18 +276,9 @@ You can use following command to run a single test method.
 
     $ python setup.py test -s <module>.<test class>.<test method>
 
-To Check for lint errors locally, install "tox" package and run following
-command.
-
-    $ pip install tox
-    $ tox -e lint_py2,lint_py3
-
-
-Beam supports running Python SDK tests using Maven. For this, navigate to root
-directory of your Apache Beam clone and execute following command. Currently
-this cannot be run from a virtual environment.
+To Check for lint errors locally, run the following command.
 
-    $ mvn clean verify -pl sdks/python
+    $ ../../gradlew lint
```

Comment:
Make it clear that ../.. is here because we assume it is being run from the sdks/python directory.

Reply:
done.